vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -163,6 +166,35 @@ func (calc *UpdatePriorityCalculator) GetSortedPods(admission PodEvictionAdmissi
 	return result
 }
 
+// GetProcessedRecommendationTargets takes a RecommendedPodResources object and returns a formatted string
+// with the recommended pod resources. Specifically, it formats the target and uncapped target CPU and memory.
+func (calc *UpdatePriorityCalculator) GetProcessedRecommendationTargets(r *vpa_types.RecommendedPodResources) string {
+	sb := &strings.Builder{}
+	for _, cr := range r.ContainerRecommendations {
+		sb.WriteString(fmt.Sprintf("%s: ", cr.ContainerName))
```

Comment:
As you're already using the string builder here you could write this without fmt.Sprintf to be more efficient.